src/pydata_sphinx_theme/theme/pydata_sphinx_theme/layout.html

```diff
@@ -84,7 +88,7 @@
       <div class="bd-sidebar-primary bd-sidebar{% if not sidebars %} hide-on-wide{% endif %}">
         {% include "sections/sidebar-primary.html" %}
       </div>
-      <main class="bd-main">
+      <main id="main-content" class="bd-main">
```

Comment:
Just double-checking that we want to skip to the "main content area" and not the "article content" on the page.

Reply:
I would say as long as we have just one, we should point to `main`. If at some point we want to add extra buttons to access all the other elements of the site we could refine it (e.g. sidebar_primary, sidebar_secondary, footer...).

What do you think ?

Reply:
sounds good - let's leave it for now and we can always move it later if there's a better option